ckanext/unhcr/fanstatic/theme.js

```diff
@@ -270,3 +270,18 @@ this.ckan.module('resource-type', function ($) {
 
   };
 });
+
+$( document ).ready(function() {
+
+  /* 
+  Update file resources from KoBo is not allowed since those files are "official" data from KoBo
+  */
+ 
+  if ($('#kobo_type_field').val() != "" ) {
+    msg = "You can change metadata but actual <b>KoBo files cannot be updated</b>.";
+    old_url = $('#field-image-url').val();
+    preserve_url = "<input type='hidden' name='url' value='" + old_url + "'>";
+    $('.image-upload').html('<div class="form-group control-full">' + msg + '</div> ' + preserve_url);
+  }
```

Comment:
Some tweaks to language and style:

![Screenshot 2021-08-25 at 13-28-14 Edit - Survey json data - RIDLTest - RIDL](https://user-images.githubusercontent.com/200230/130782865-d6d1ef06-d1eb-4448-9f68-d41780867cec.png)


```suggestion

  /* 
  Update file resources from KoBo is not allowed since their files are always directly imported from KoBo
  */
  let kobo_type = $('#kobo_type_field').val();
  if (kobo_type !== undefined && kobo_type != "") {
    // TODO: link to where you can re-import them
    msg = "You can change the resource metadata but the actual data files are imported from Kobo.";
    old_url = $('#field-image-url').val();
    preserve_url = "<input type='hidden' name='url' value='" + old_url + "'>";
    $('.image-upload').html('<div class="alert alert-warning">' + msg + '</div> ' + preserve_url);
  }
```

This is to cover the case where the input field is not present in the page and has a value:

```
if (kobo_type !== undefined && kobo_type != "") 
```